src/cadence/mainnet/getNFTs.cdc

```diff
@@ -3551,3 +3553,41 @@ pub fun getSwaychainNFT(owner: PublicAccount, id: UInt64): NFTData? {
         }
     )
 }
+
+// https://flow-view-source.com/mainnet/account/0x33f44e504a396ba7/contract/MyNFT
+pub fun getSkyhorborNFT(owner: PublicAccount, id: UInt64): NFTData? {
+    let contract = NFTContract(
+        name: "Skyharbor.app",
+        address: 0x33f44e504a396ba7,
+        storage_path: "/storage/MyNFTCollection",
+        public_path: "/public/MyNFTCollection",
+        public_collection_name: "MyNFT.CollectionPublic",
+        external_domain: ""
+    )
+
+    let col = owner.getCapability(/public/MyNFTCollection)
+        .borrow<&{MyNFT.CollectionPublic}>()
+    if col == nil { return nil }
+
+    let nft = col!.borrowReward(id: id)
```

Comment:
Seeing this error on running the test:
258 |                 case "MyNFT": d = getSkyhorborNFT(owner: owner, id: id)
    |                                   ^^^^^^^^^^^^^^^ not found in this scope

Reply:
Applied the fix and updated files on my repo. 

Please review it.

Reply:
Still seeing errors, can you please run tests before submitting the PR:
    --> d247108fdcd05636060b63a02b5c1758fbdbaee8f5ef948c1cfa34850c7f7233:3854:14
     |
3854 |     let nft = col!.borrowEntireNFT(id: id)
     |               ^^^^

error: value of type `&MyNFT.NFT` has no member `name`
    --> d247108fdcd05636060b63a02b5c1758fbdbaee8f5ef948c1cfa34850c7f7233:3861:20
     |
3861 |         title: nft!.name,
     |                     ^^^^ unknown member

error: value of type `&MyNFT.NFT` has no member `editionNumber`
    --> d247108fdcd05636060b63a02b5c1758fbdbaee8f5ef948c1cfa34850c7f7233:3867:23
     |
3867 | 		"editionNumber": nft.editionNumber,
     |                        ^^^^^^^^^^^^^ unknown member

Reply:
> 

Hi,

I have fixed the name attribute issue getting in getNfts.cdc file.

But while running below test case command I'm getting error like ==> **Command Error: error parsing script arguments: invalid character 't' looking for beginning of object key string**  

Am I passing something wrong in these command. I have tried with below 2 commands  Please suggest on this. Thanks

MyNFT command:
flow scripts execute getNFTs.cdc --args-json '[{ "type": "Address", "value": "0x5affd8c030ae5761" }, { "type": "Dictionary", "value": [{ "key": { "type": "String", "value": "MyNFT" }, "value": { "type": "Array", "value": [{ "type": "UInt64", "value": "128" }] } }] }]' --network mainnet

![PS_Scripts](https://user-images.githubusercontent.com/1484621/171861426-dfab98c1-b40e-4eb9-bb2b-5157a7bc084f.png)


Sample command provided in the repo note:
flow scripts execute getNFTs.cdc --args-json '[{ "type": "Address", "value": "0x9eef2e4511390ce4" }, { "type": "Dictionary", "value": [{ "key": { "type": "String", "value": "Gaia" }, "value": { "type": "Array", "value": [{ "type": "UInt64", "value": "1129" }] } }] }]' --network mainnet

Reply:
Hi,

Please review it once and suggest.

Reply:
Hi, Please check it once on the test case execution issue and suggest. So we can proceed further and complete it. Thanks in advance.